City-of-Helsinki/drupal-asuntotuotanto
# ASU-398 admin theme

How to test
- `make fresh`
- Log in and go to create a project: https://asuntotuotanto.docker.sh/fi/node/add/project
- Check that the layout and tabs are according to [design](https://share.goabstract.com/a8f8852e-c04e-480c-b75c-cf16ede2a7c9?collectionLayerId=20165b61-2a39-4d12-9366-8230790e8abe&mode=build&nonce=20165b61-2a39-4d12-9366-8230790e8abe&sha=7d6857e2320de459209b0604b7528f4a849dcda0)
